client/src/www/app/outline_server_repository/server.ts

```diff
@@ -135,7 +136,7 @@ export class OutlineServer implements Server {
   }
 }
 
-function parseTunnelConfigJson(responseBody: string): TunnelConfigJson | null {
+export function _parseTunnelConfigJson(responseBody: string): TunnelConfigJson | null {
```

Comment:
Is it OK to export a function starting with `_`?

Reply:
I was trying to keep the visibility protected, but the style guide doesn't actually allow that, but I found a better [recommendation](go/typescript-testing#export-visibility).